code/modules/modular_armor/attachables.dm

```diff
@@ -50,7 +50,7 @@
 
 /obj/item/armor_module/attachable/better_shoulder_lamp/mark1
 	name = "\improper Mark 1 Baldur Light Amplification System"
-	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Substantially increases the power output of the Jaeger Combat Exoskeleton's mounted flashlight. Slows you down minorly."
```

Comment:
unrelated changes

Reply:
Yes its the other one put together. I dont know how much a single addition hurts the total so I can close the other one if you want and keep them together. Title the whole thing description fixes.